Mathlib/Algebra/Parity.lean

```diff
@@ -347,18 +350,22 @@ theorem odd_one : Odd (1 : α) :=
   ⟨0, (zero_add _).symm.trans (congr_arg (· + (1 : α)) (mul_zero _).symm)⟩
 #align odd_one odd_one
 
+@[simp] lemma Even.add_one (h : Even m) : Odd (m + 1) := h.add_odd odd_one
+
+@[simp] lemma Even.one_add (h : Even m) : Odd (1 + m) := h.odd_add odd_one
+
 @[simp]
 theorem odd_two_mul_add_one (m : α) : Odd (2 * m + 1) :=
```

Comment:
Seeing that we have this lemma is enough to convince me that your new lemmas are fine.